lib/src/components/card/card.dart

```diff
@@ -73,15 +83,19 @@ class SkawaCardContentComponent {
   }
 }
 
-/// Header component for a [SkawaCardComponent]
+/// Header component for a [SkawaCardComponent].
 ///
 /// __Example:__
 ///
 ///     <skawa-card>
 ///       <skawa-card-header statusColor="rgb(255,0,0)">
-///
 ///       </skawa-card-header>
 ///     </skawa-card>
+///
+/// __Inputs:__
+///   - `statusColor: String` -- The color of the box-shadow-top. Accept only rgb() or rgba() formats. Defaults to transparent.
+///   - `backgroundColor: String` -- ???
```

Comment:
Missing documentation line. We can probably remove this input unless we find a good use-case to support it.